### PR TITLE
Import v5 csvs

### PR DESCRIPTION
**Pull request by sduckett**
The final of three 5.0 CSVs, in this PR we finally load the data into a database. Validations (outside of those included in the data spec) will happen in later stories.

[Pivotal card](https://www.pivotaltracker.com/story/show/118434215)

### CODE REVIEW

**Comment by danielglauser** [comment]
Are all the migrations the same as [this](https://github.com/votinginfoproject/data-processor/pull/159) PR?

**Comment by sduckett** [comment]
@danielglauser Yes, the code here is branched from PR 159, where the tables were created.

File: test/vip/data_processor/validation/csv_test.clj [hunk 1]
```diff
@@ -152,11 +152,7 @@
           three-point-0-pipeline (get version-pipelines "3.0")]
       (is (= three-point-0-pipeline
              (take (count three-point-0-pipeline) (:pipeline out-ctx))))))
-  (testing "stops with unsupported version for 5.0 feeds"
```
Comment (danielglauser) [inline test/vip/data_processor/validation/csv_test.clj]: Would love to see some 5.0 positive and negative tests.

Comment (sduckett) [inline test/vip/data_processor/validation/csv_test.clj]: @danielglauser

> Would love to see some 5.0 positive and negative tests.

 Does e6519474bcb2004e6836633794e991633fc219f7 address this in a way that makes you happy?

Comment (sduckett) [inline test/vip/data_processor/validation/csv_test.clj]: Also, this is loading the data _without_ any type of validations; we'll be covering input validation and relations between tables in upcoming stories like [this](https://www.pivotaltracker.com/story/show/118947361) and [that](https://www.pivotaltracker.com/story/show/108896232), etc.

**Comment by danielglauser** [comment]
Sounds good @sduckett, LGTM

**Pull request merged**